src/components/forms/profile/user-address-form.tsx

```diff
@@ -94,6 +116,30 @@ const UserAddressForm: React.FC<{
     street_address_2: "",
     primary: false,
   });
+  const [addresses, setAddresses] = React.useState<Address[]>([]);
```

Comment:
We need to use that to get the addresses. If we don't, we don't have any other way to get the value of the cache.